vc-verifier/js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vc-verifier-js",
+  "version": "0.0.1",
+  "description": "JavaScript Library for VC Verification",
+  "repository": "https://github.com/mosip/vc-verifier",
+  "main": "src/CredentialsVerifier.js",
+  "scripts": {
+    "test": "node --experimental-vm-modules ./node_modules/.bin/jest ",
+    "start": "NODE_OPTIONS='--experimental-specifier-resolution=node' node src/CredentialsVerifier.js",
```

Comment:
This is not really required in our case as its a library.

Reply:
Let's keep this for now till the time we are developing. It will ease out the pain of the develops to run the code each time. We can remove it later.